tools/tertiary-analysis/cell-types-analysis/ct_get_tool_perf_table.xml

```diff
@@ -20,6 +20,9 @@
         #if $parallel
         --parallel "${parallel}"
         #end if
+        #if $include-sem-siml
+        --include-sem-siml "${include_sem_siml}"
```

Comment:
although beyond this PR, I normally don't like in a CLI these options that ask for true or false... they should be flags rather than options. More writing for the user!